airbyte-workers/src/main/java/io/airbyte/workers/internal/AirbyteMessageTracker.java

```diff
@@ -129,8 +131,10 @@ private void handleSourceEmittedRecord(final AirbyteRecordMessage recordMessage)
    * correctly.
    */
   private void handleSourceEmittedState(final AirbyteStateMessage stateMessage) {
+    log.info("source emitted state message");
```

Comment:
I wonder if logging a message per StateMessage is a bit too noisy for info. Does it make sense to have that in debug?